client/stylesheets/_modal.scss

```diff
@@ -141,7 +141,8 @@
 /*~~ Button End ~~*/
 
 .notifMainWrap h2 {
-	margin-top: 50px !importants;
+	margin-top: 50px !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered font-weight, margin-top